tools/nebula-flink/src/test/java/org/apache/flink/connector/nebula/sink/AbstractNebulaOutPutFormatTest.java

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package org.apache.flink.connector.nebula.sink;
+
+import junit.framework.TestCase;
+import org.apache.flink.connector.nebula.connection.NebulaClientOptions;
+import org.apache.flink.connector.nebula.connection.NebulaConnectionProvider;
+import org.apache.flink.connector.nebula.connection.NebulaGraphConnectionProvider;
+import org.apache.flink.connector.nebula.statement.ExecutionOptions;
+import org.apache.flink.types.Row;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class AbstractNebulaOutPutFormatTest extends TestCase {
+    @Test
+    public void testWrite() throws IOException {
+        List<String> cols = new ArrayList<>();
+        cols.add("name");
+        cols.add("age");
+        ExecutionOptions executionOptions = new ExecutionOptions.ExecutionOptionBuilder()
+                .setDataType("VERTEX")
```

Comment:
why don't use enum

Reply:
you are right, test should also be standardized.